### PR TITLE
Add release notes for 0.233.1

### PR DESCRIPTION
**Pull request by caithagoras**
```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.233.1.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+===============
+Release 0.233.1
+===============
+
+General Changes
+_______________
+* Fix an issue where queries with both ``COALESCE`` and ``FULL OUTER JOIN`` may produce incorrect
```
Comment (bhhari) [inline presto-docs/src/main/sphinx/release/release-0.233.1.rst]: @kaikalur  was related to bucketing, I believe this was from @yingsu00

**Comment by kaikalur** [comment]
== RELEASE NOTES ==
Fixed bucket pruning when the predicate is IS NULL and the table actually has rows with that value which we were previously not considering.

File: presto-docs/src/main/sphinx/release/release-0.233.1.rst (hunk 1 above)
Comment (kaikalur) [reply presto-docs/src/main/sphinx/release/release-0.233.1.rst]: > @kaikalur was related to bucketing, I believe this was from @yingsu00

== RELEASE NOTES ==
Fixed bucket pruning when the predicate is IS NULL and the table actually has rows with that value which we were previously not considering.

Comment (caithagoras) [reply presto-docs/src/main/sphinx/release/release-0.233.1.rst]: @kaikalur Gotcha. Can you rephrase this to be user-facing. User do not understand what is "buck pruning".

Comment (mayankgarg1990) [reply presto-docs/src/main/sphinx/release/release-0.233.1.rst]: Bucketing is a Hive fix, so this should go to `Hive changes`, right?

`Fix an issue where queries with the predicate `IS NULL` on bucketed columns would always return 0 results.` ?

Comment (bhhari) [reply presto-docs/src/main/sphinx/release/release-0.233.1.rst]: Fix incorrect results returned when selecting from a bucketed table when bucket predicate is NULL

**Comment by caithagoras** [comment]
@mayankgarg1990 "query would always return 0 results" is not true. Changing to "produce incorrect results".

Warning added to 0.231, 0.232 and 0.233 release notes.
